CMSgov/mint-app
# [MINT-1727] Model Plan Change History

# EASI-2771, [MINT-1727](https://jiraent.cms.gov/browse/MINT-1727)

## Description
This PR represents the entirety of the model plan change history feature. This work has previously gone through PRs to get to feature, the goal of this PR is to do a high level review.

Please take your time reviewing figma, the ticket in JIRA, and the code.



## How to test this change
1. Bring up the backend `scripts/dev up:backend:debug`
2. Bring up the front end `yarn && yarn start`
3. Run the db seed utility tool in interactive mode `go run ./cmd/dbseed/*go` alternatively, run `go run ./cmd/dbseed/*go translate` whenever you want data to translate
4. Go through the application and make some changes (can also translate seed data)
5. Explore the front end change history pages to make sure they make sense

Please also take your time to go through the architecture. 
Areas to focus on
1. Shared Mappings
2. Translation Job
3. Translated Audit Package
4. Database table updates
5. Frontend field groupings
6. Documentation

- 

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
